WPThemeReview/Tests/Templates/ReservedTemplateNameTests/page-contact.inc

```diff
@@ -0,0 +1,4 @@
+<?php
+/* Template Name: Contact page template */
+
+// code goes here
```

Comment:
Might be an idea to have some dummy inline HTML with embedded PHP here to safe-guard that only one error will be thrown per file.